packages/drivers/iframe-driver/src/outerDocumentServiceFactory.ts

```diff
@@ -165,8 +165,8 @@ export class DocumentServiceFactoryProxy implements IDocumentServiceFactoryProxy
             read: async (id) => {
                 return storage.read(id);
             },
-            write: async (root, parents, message, ref) => {
```

Comment:
I'd remove all driver related (not specific to branches) changes out to separate PR

Reply:
Reverted in latest